.github/workflows/check-external-links.yaml

```diff
@@ -0,0 +1,40 @@
+name: Check external links
+
+on: workflow_dispatch
+
+jobs:
+  check-links:
+    name: Check for broken external links
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.1
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4.0.4
+        with:
+          cache: npm
+          check-latest: true
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Restore build
+        uses: actions/cache/restore@v4.1.1
+        with:
+          key: build-cache-${{ runner.os }}-${{ github.sha }}
+          path: build
+
+      - name: Run task
+        id: task
+        run: ${{ matrix.task.run }}
+        continue-on-error: ${{ matrix.task.continue-on-error || false }}
+
```

Comment:
**issue** Given there's no `matrix` of commands to run in parallel, we can remove that step from the workflow.

```suggestion
```